cts/cts-cli.in

```diff
@@ -1860,15 +1860,23 @@ if [ $num_errors -ne 0 ]; then
     echo "$num_errors tests failed; see output in:"
```

Comment:
Thankfully we already have $failed that tells us whether any output was different, so we can skip the "see output in" if $failed is 0